styles/lobste.rs/catppuccin.user.css

```diff
@@ -0,0 +1,153 @@
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('lobste.rs') {
+  @media (prefers-color-scheme: dark) {
+    html.color-scheme-system {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: light) {
+    html.color-scheme-system {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+
+  html.color-scheme-light {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  html.color-scheme-dark {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    background-color: @base;
+
+    --palette-bg: @base;
+    --palette-fg: @text;
+
+    --color-fg: @text;
+    --color-fg-shape: @surface2;
+
+    --color-tag-bg: fade(@yellow, 20%);
+    --color-tag-border: fade(@yellow, 20%);
+
+    --color-lobsters-tag-special-bg: fade(@red, 30%);
+    --color-lobsters-tag-special-border: fade(@red, 30%);
+
+    --color-tag-media-bg: fade(@blue, 20%);
+    --color-tag-media-border: fade(@blue, 30%);
```

Comment:
Just curious what these variables theme specifically and how it looks?

Reply:
Those are the post tags! This is how they look styled vs unstyled:

![Screenshot 2024-11-18 at 11-21-32 Tags Lobsters](https://github.com/user-attachments/assets/5b5ea24e-35d6-4d8e-8a88-1d20b81f7152)

![Screenshot 2024-11-18 at 11-21-55 Tags Lobsters](https://github.com/user-attachments/assets/a45d6b28-fc41-4fe0-9f6d-a032bb106aca)